src/App.tsx

```diff
@@ -1,29 +1,23 @@
-import { Component } from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
+import { Movie } from './react-app-env';
```

Comment:
Your movie is not clearing from preview after added to list of movies on the left. It should clear as soon as you click add to the list button.